content/devs.json

```diff
@@ -183,5 +183,39 @@
         "url": "mailto:tj.andoy@gmail.com"
       }
     ]
+  },
+  {
+    "avatar": "http://adrianriosweb.com/wordpress_react/wp-content/uploads/2020/10/me.min_.jpg",
```

Comment:
Your profile is not loading on the website because the image is not a secured element.

Please try to host your image on a secured cdn or you may try this:
https://imgbb.com/

Please also fix the conflicting files. Thanks.
![Screen Shot 2020-10-26 at 10 08 26 PM](https://user-images.githubusercontent.com/45030270/97182675-cb3b4a00-17d7-11eb-9bdd-daff1b57fb71.png)

Reply:
Hi, I updated the avatar url, hosted using imgbb.com, also fixed conflicts.